components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp

```diff
@@ -3812,7 +3815,7 @@ function doValidation() {
             <td>
                 <div class="sectionCheckbox">
                     <input id="authnResponseSigned" name="authnResponseSigned"
-                           type="checkbox" <%=authnResponseSignedChecked%>/>
+                           type="checkbox" <%=Encode.forHtmlAttribute(authnResponseSignedChecked)%>/>
```

Comment:
The variable value is hardcoded. Further this is not an attribute. The variable fully represents the attribute key and value here. So no need to encode.